spec/setup_acceptance_node.pp

```diff
@@ -1,48 +1,9 @@
-class { 'foreman::repo':
-  repo => 'nightly',
-}
-
-stdlib::ensure_packages(['podman'])
-
-$container_file = "[Unit]
-Description=Advisor Engine
-After=network-online.target
-[Container]
-Image=registry.access.redhat.com/ubi9/python-312
-Exec=python /usr/local/bin/python-https-server
-Network=host
-Volume=/usr/local/bin/python-https-server:/usr/local/bin/python-https-server
-[Service]
-Restart=always
-[Install]
-WantedBy=default.target
-"
-
-$python_server = "
-from http.server import HTTPServer, SimpleHTTPRequestHandler
-import ssl
-import os
-
-context = ssl.SSLContext(ssl.PROTOCOL_TLS_SERVER)
-context.load_cert_chain(certfile='/opt/app-root/src/cert.pem', keyfile='/opt/app-root/src/key.pem')
-context.check_hostname = False
-
-port=int(os.environ.get('IOP_ADVISOR_ENGINE_PORT', 8000))
-
-with HTTPServer(('localhost', port), SimpleHTTPRequestHandler) as httpd:
-    httpd.socket = context.wrap_socket(httpd.socket, server_side=True)
-    httpd.serve_forever()
-"
-
-file { '/etc/containers/systemd/iop-advisor-engine.container':
-  ensure  => present,
-  content => $container_file,
-  require => Package['podman'],
+package { 'podman':
```

Comment:
Cheater.

Reply:
I didn't know how else to resolve this for now. I suppose this is a design flaw in this module that it both installs podman and relies upon it being installed.

Reply:
Unless podman is already present through some other means? hrmm. Our other options being...

1. switch to https://github.com/voxpupuli/puppet-quadlets/blob/master/manifests/quadlet.pp which doesn't seem to have this issue
2. disable the service management in the quadlet definition and add our own

Reply:
The `if podman version` is around the *whole* block: https://github.com/southalc/podman/blob/master/manifests/quadlet.pp#L62
So disabling the service management won't be sufficient :/

So yeah, 1 or 3 "use an epp file for the quadlet definition and the module only for the secret"